modules/quant_loader.py

```diff
@@ -7,28 +7,20 @@
 import modules.shared as shared
 
 sys.path.insert(0, str(Path("repositories/GPTQ-for-LLaMa")))
-from llama import load_quant
 
 
 # 4-bit LLaMA
-def load_quantized_LLaMA(model_name):
-    if shared.args.load_in_4bit:
-        bits = 4
+def load_quantized(model_name, model_type):
+    if model_type == 'llama':
+        from llama import load_quant
+    elif model_type == 'opt':
+        from opt import load_quant
     else:
-        bits = shared.args.gptq_bits
+        print("Unknown pre-quantized model type specified. Only 'llama' and 'opt' are supported")
+        exit()
 
     path_to_model = Path(f'models/{model_name}')
-    pt_model = ''
-    if path_to_model.name.lower().startswith('llama-7b'):
```

Comment:
Removing this block breaks compatibility with folder names like `llama-7b-hf` by decapoda.

Reply:
I moved the code back